### PR TITLE
Geting JWST/NIRCAM Wide-Field Slitless Spectroscopy Started

### PR DESCRIPTION
**Pull request by jhennawi**
This very short PR to the JWST branch provides a starting implementation for JWST slitless spectroscopy with NIRCAM. Since the jwst branch has already been reviewed, it makes sense for this short PR to be reviewed by the same reviewers.

### CODE REVIEW

File: pypeit/spectrographs/jwst_nircam.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+class JWSTNIRCamSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle MMT/Blue Channel specific code
+    """
+    ndet = 2
+    name = 'jwst_nircam'
+    header_name = 'jwst_nircam'
+    telescope = telescopes.JWSTTelescopePar()
+    camera = 'NIRCAM'
+    supported = True
```
Comment (profxj) [inline pypeit/spectrographs/jwst_nircam.py]: I don't think we really mean this.
Set to `False`

**Review by profxj — APPROVED** [review]
One nit

File: pypeit/spectrographs/jwst_nircam.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+from pypeit.core import framematch
+from pypeit.par import pypeitpar
+from pypeit.spectrographs import spectrograph
+from pypeit.core import parse
+from pypeit.images import detector_container
+from IPython import embed
+
+class JWSTNIRCamSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle MMT/Blue Channel specific code
```
Comment (debora-pe) [inline pypeit/spectrographs/jwst_nircam.py]: It would be good to change this description

**Review by debora-pe — APPROVED** [review]
Looks good.
The CI tests failure I believe are due to some problems with the `jwst` branch.

File: pypeit/spectrographs/jwst_nircam.py (hunk 2 above)
Comment (jhennawi) [reply pypeit/spectrographs/jwst_nircam.py]: Fixed.

File: pypeit/spectrographs/jwst_nircam.py (hunk 1 above)
Comment (jhennawi) [reply pypeit/spectrographs/jwst_nircam.py]: done.

**Pull request merged**